### PR TITLE
Add developer/maintainer section and add flatpak repo docs

### PR DESCRIPTION
**Pull request by jstebbins**

### CODE REVIEW

File: source/en/latest/table-of-contents.markdown [hunk 1]
```diff
@@ -138,3 +138,8 @@ Table of contents
   - [DRC](technical/audio-drc.html)
   - [Mixdown](technical/audio-mixdowns.html)
 - Advanced preferences
+
+## Developer/Maintainer documentation
```
Comment (bradleysepos) [inline source/en/latest/table-of-contents.markdown]: Can probably shorten this to Developer documentation.

File: source/en/latest/developer/flatpak-repo.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Maintaining a HandBrake flatpak repository
+==========================================
+
+## Creating a new empty flatpak repository
+This would be done once to set up the repo users update their HandBrake
+flatpaks from. This repo gets served via http to users.
+
+Initialize a new repository  
+`ostree init --mode=archive-z2 --repo=<repo-dir>`
```
Comment (bradleysepos) [inline source/en/latest/developer/flatpak-repo.markdown]: I would probably separate each of these pairs with an empty space, and indent the code block instead of using backticks.

Initialize a new repository

    ostree init --mode=archive-z2 --repo=<repo-dir>

**Comment by bradleysepos** [comment]
Nits above. Looking good!

File: source/en/latest/developer/flatpak-repo.markdown (hunk 2 above)
Comment (jstebbins) [inline source/en/latest/developer/flatpak-repo.markdown]: I don't really like how much blank space this adds and how the example command is centered between things.  I like it better nudged up closer to the text that describes what it's doing.  I actually did something like this initially and then started asking you questions about how to do line breaks because I didn't like it.

File: source/en/latest/table-of-contents.markdown (hunk 1 above)
Comment (jstebbins) [inline source/en/latest/table-of-contents.markdown]: Ok

File: source/en/latest/developer/flatpak-repo.markdown (hunk 2 above)
Comment (bradleysepos) [inline source/en/latest/developer/flatpak-repo.markdown]: Certainly don't mean to bikeshed but line breaks are generally discouraged except in special cases. I pushed another style update that should allow you to use the suggested markup while remedying the display issue.

Comment (jstebbins) [inline source/en/latest/developer/flatpak-repo.markdown]: Umm, this *is* a special case.  It's a code block preceded by a description of that code block.  The 2 things are associated, but must be on separate lines.  It makes sense for the code block to be physically closer to the line that describes it than to the things below for readability.

Comment (bradleysepos) [inline source/en/latest/developer/flatpak-repo.markdown]: Which is why I moved the preformatted code block physically closer, and indented it. :laugh:

Comment (jstebbins) [inline source/en/latest/developer/flatpak-repo.markdown]: Ah, now that's much better :smiling_imp:

**Comment by bradleysepos** [comment]
Squash and ship it!
